python/histone_utility.py

```diff
@@ -0,0 +1,355 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Jul 25 11:28:01 2019
+
+@author: Marc Di Meo
+"""
+
+import mygene
+import pandas as pd
+import numpy as np
+from sklearn import preprocessing
+import os
+from scipy.stats.stats import pearsonr
+
+
+def read_csv_df(csv_file):
+    """Read in a csv file and convert it to a dataframe. 
+    
+    This function will remove expression values of zero and replace it with NaN
+    
+    INPUT
+        file: path to desired file
+        
+    OUTPUT
+        data: data with empty values as NaN    
+        
+    """
+    data = pd.read_csv(csv_file, delim_whitespace=True)
+    data = data.replace(0, np.nan)
+    return data
+
+
+def change_filepath(path):
+    """Change file path to look for other files
+    
+    INPUT 
+        path: new directory you would like to look for file
+        
+    """
+    
+    os.chdir(path)  
+
+def remove_columns(data, list_columns):    
+    """Remove unnecssary dataframe columns
+    
+    INPUT
+        data: dataframe with unnecessary columns
+        list_columns: list of columns to remove from the dataframe
+    
+    OUTPUT
+        data: dataframe without the columns you wished to remove
+        
+    """
+    
+    data = data.drop(columns, axis=1)
+    return data
+
+
+def remove_duplicate_entries(items_to_remove): 
+    """This function will remove duplicated elements from a list
+    
+    INPUT
+        list_of_items: list which you would like to remove any duplicated items
+    
+    OUTPUT
+        final_list: list with no repeating elements
+        
+    """
+    final_list = [] 
+    for num in items_to_remove: 
+        if num not in items_to_remove: 
+            final_list.append(num) 
+    return final_list
+
+
+def gene_conversion(gene_list, scopes, fields, species):
+    """This serves as a way to convert gene ids into gene  
+    symbols which is much easier to work with. 
+    
+    It is important to note that the gene_list may require manipulation 
+    as mygene works differently for each scope you use. The manipulation will 
+    depend on the scope being used. For example 'ensembl.gene' ids look similar
+    to ENSG00000000003 however they can have decimals (ENSG00000000003.3)
```

Comment:
This sounds like a simple fix. Can you write a conditional where if they choose ensembl gene identifiers, it turns that float value to an int?